storefront/modules/catalog/services/CategoryService.ts

```diff
@@ -1,11 +1,14 @@
 import { Category } from '../models/Category';
+import apiClientService from '@/common/services/ApiClientService';
+
+const baseUrl = '/api/product/storefront/categories';
 
 export async function getCategories(): Promise<Category[]> {
-  const response = await fetch('/api/product/storefront/categories');
+  const response = await apiClientService.get(baseUrl);
   return await response.json();
 }
 
 export async function getCategory(id: number) {
-  const response = await fetch('/api/product/storefront/categories/' + id);
+  const response = await apiClientService.get(baseUrl + id);
```

Comment:
`apiClientService.get(baseUrl + id);`
Is the url missing the slash "/"?

Reply:
Yes, I've configured the url in the newest code to resolve the problem